pkg/cli/common/util.go

```diff
@@ -70,3 +73,27 @@ func LoadFileIntoVar(filePath string, destContent *string) error {
 
 	return nil
 }
+
+// WriteConfigFile writes new content of the config file.
+// If the file does not exist, it is created at default locatio.
+// If the file does not exist, it is created at default location
+// TODO temporary solution until upstream https://github.com/spf13/viper/issues/433 is fixed
+func WriteConfigFile() error {
+	cf := viper.ConfigFileUsed()
+
+	if cf == "" {
+		fullname := ConfigFileName + "." + ConfigFileType
+		if dirname, err := os.Getwd(); err == nil {
+			cf = filepath.Join(dirname, ConfigHomeSubdir, fullname)
+		}
+		if dirname, err := os.UserHomeDir(); err == nil {
+			cf = filepath.Join(dirname, ConfigHomeSubdir, fullname)
+		}
+		fmt.Printf("New config file to be created at %s\n", cf)
```

Comment:
nit: even if it's printed before, could you make this say "FuseML configuration file created at" instead ? I.e. to state a fact, not an intention.

Reply:
Sure